src/boards/rainbow_esp.cpp

```diff
@@ -365,35 +373,79 @@ void BrokeStudioFirmware::processBufferedMessage() {
 				this->writeFile(this->working_path, this->working_file, this->files[working_path][working_file].size(), this->rx_buffer.begin() + 3, this->rx_buffer.begin() + message_size + 1);
 			}
 			break;
-		case n2e_cmds_t::GET_FILE_LIST:
-			UDBG("RAINBOW BrokeStudioFirmware received message GET_FILE_LIST\n");
+		case n2e_cmds_t::FILE_COUNT:
```

Comment:
This command is not in the doc

Reply:
Oops, forgot to commit and push the doc update... Done.